spec/unit/resource/api600/synergy/os_deployment_plan_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::API500::Synergy::OSDeploymentPlan do
+RSpec.describe OneviewSDK::API600::Synergy::OSDeploymentPlan do
   include_context 'shared context'
 
   it 'inherits from OneviewSDK::API300::Synergy::OSDeploymentPlan' do
```

Comment:
inherits from 500

Reply:
I have inherited the lib file from 300 to avoid one more level of inheritance, as there is no change.

Reply:
Ok. LGTM 👍